packages/components/src/internal/components/forms/QueryFormInputs.tsx

```diff
@@ -59,7 +59,7 @@ interface QueryFormInputsProps {
     queryColumns?: OrderedMap<string, QueryColumn>;
     queryInfo?: QueryInfo;
     lookups?: Map<string, number>;
-    onChange?: Function;
+    onFileChange?: (fileMap: Record<string, File>) => void;
```

Comment:
renaming this to `onFileChange` makes sense, as we discussed. but is this still needed anymore after the changes you made so that we don't need to handle the fileMap for the separate instances / usages?

Reply:
Yes, I went back and forth. I'll remove it.